packages/loopring_v3/package.json

```diff
@@ -77,6 +77,8 @@
     "pretty-quick": "^1.11.1",
     "solc": "^0.5.11",
     "solidity-coverage": "^0.5.11",
+    "solium": "^1.2.5",
```

Comment:
`ethlint` is the correct package for solium:
- https://github.com/duaraghav8/Ethlint#install